docs/public/SUMMARY.md

```diff
@@ -2,6 +2,7 @@
  https://rust-lang.github.io/mdBook/format/summary.html -->
 # Documents in public document tree
 
+- [Zonemaster documentation](README-for-doc.zonemaster.net.md)
```

Comment:
This will be strange in the Github context, won't it? Especially since the file does not exist in that directory.

Reply:
I think it should be
```
- [Zonemaster documentation](README.md)
```
in SUMMARY.md and the script will do a
```
cp ../README-for-doc.zonemaster.net.md README.md
```
from the docs/public directory.